tests/rbd_mirror/rbd_mirror_reconfigure_secondary_cluster.py

```diff
@@ -0,0 +1,114 @@
+"""Test case covered -
+    CEPH-9477- Secondary cluster permanent failure,
+    Recreate the secondary cluster and Re-establish mirror with newly created cluster as secondary.
+
+    Pre-requisites :
+    1. Cluster must be up and running with capacity to create pool
+    (At least with 64 pgs)
+    2. We need atleast one client node with ceph-common package,
+    conf and keyring files
+
+    Test case flows:
+    1) After site-b failure, bring up new cluster for secondary
+    2) Create a pool with same name as secondary pool
+    3) Perform the mirroring bootstrap for cluster peers
+    4) copy and import bootstrap token to peer cluster
+    5) verify peer cluster got added successfully after failover
+    6) verify all the images from secondary mirrored to primary
+"""
+import datetime
+import time
+
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """
+    Verification of rbd mirroring failover scenario on secondary site,
+    This module configures the mirroring to newly created cluster.
+
+    Args:
+        kw: test data
+        Example::
+            config:
+                poolname: pool_failback
+    Returns:
+        int: The return value - 0 for success, 1 for failure
+    """
+    try:
+        log.info("Running test CEPH-9477")
+
+        config = kw.get("config")
+        poolname = config.get("poolname")
+
+        mirror3 = rbdmirror.RbdMirror(
+            kw.get("ceph_cluster_dict").get("ceph-rbd3"), config
+        )
+
+        mirror1 = rbdmirror.RbdMirror(
+            kw.get("ceph_cluster_dict").get("ceph-rbd1"), config
+        )
+
+        rbd1, _, rbd3 = [
+            Rbd(**kw, req_cname=cluster_name)
+            for cluster_name in kw["ceph_cluster_dict"].keys()
+        ]
+
+        kw["peer_mode"] = kw.get("peer_mode", "bootstrap")
+        kw["rbd_client"] = kw.get("rbd_client", "client.admin")
+        kw["failback"] = True
+
+        log.info("Re-establishing mirror with newly created cluster as Primary")
+
+        # # remove the failed cluster from mirror peers
```

Comment:
please remove double # for all comments.

Reply:
done.